CDCgov/prime-reportstream
# [16143] Implement Convert Step Updates

This PR contains coding changes for user story [16144](https://github.com/CDCgov/prime-reportstream/issues/16143).

Test Steps:
1. Debug unit tests or integration tests locally with breakpoints set in affected code to verify functionality.
2. POST a message through Postman (or equivalent) locally and observe the log files.
3. Connect dev env to testing app insights and verify events show up properly.

## Changes
- Modified FHIRConverter and associated tests to:
  - Only add extensions if message is an ELR.
  - Send an ITEM_TRANSFORMED Azure event.


## Checklist

### Testing

### Process
No changes.
